plugin/pulpcore/plugin/changeset/iterator.py

```diff
@@ -146,15 +147,19 @@ class DownloadIterator(Iterable):
     Iterate the content artifacts and yield the appropriate download object.
     When downloading is deferred or the artifact is already downloaded,
     A NopDownload is yielded.
+
+    Attributes:
+        input (iterable): An iterable of PendingArtifact.
+        deferred (bool): Downloading is deferred.
     """
 
-    def __init__(self, content, deferred=False):
+    def __init__(self, artifacts, deferred):
         """
         Args:
-            content (Iterable): An Iterable of RemoteContent.
-            deferred (bool): When true, downloading is deferred.
+            artifacts (iterable): An iterable of PendingArtifact.
+            deferred (bool): Downloading is deferred.
         """
-        self.content = content
+        self.input = artifacts
```

Comment:
It seemed clearer in the code to operate on the collection of content that is the _input_ to the iterator.  I don't feel strongly.